_data/menus.yml

```diff
@@ -11,6 +11,9 @@ main:
   - name: "Contact"
     url: "/contact/"
     weight: 5
+  - name: "Blog"
+    url: "/blog/"
+    weight: 6
```

Comment:
It works great. however, optionally you can move this above Contact. So Contact stay as the last item.

Reply:
Thanks. I have set Contact as the last item on the menu.

Reply:
Great. Feel free to then close this item as 'resolved' in github.